provisioning/Samples/device/SymmetricKeyWithOperationalCertificateSample/ProvisioningDeviceClientSample.cs

```diff
@@ -0,0 +1,182 @@
+
+                if (result.IssuedClientCertificate == null)
+                {
+                    _logger.LogError($"Expected operational certificate was not returned by DPS, so exiting this sample.");
+                    return;
+                }
+
+                // This sample uses openssl to generate the pfx certificate from the issued operational certificate and the previously created ECC P-256 keypair.
+                // This certificate will be used when authneticating with IoT hub.
+                _logger.LogInformation("Creating an X509 certificate from the issued operational certificate...");
+                using X509Certificate2 clientCertificate = GenerateOperationalCertificateFromIssuedCertificate(result.RegistrationId, result.IssuedClientCertificate);
+                IAuthenticationMethod auth = new DeviceAuthenticationWithX509Certificate(result.DeviceId, clientCertificate);
+
+                _logger.LogInformation($"Testing the provisioned device with IoT Hub...");
+                using DeviceClient iotClient = DeviceClient.Create(result.AssignedHub, auth, _parameters.TransportType);
+
+                _logger.LogInformation("Sending a telemetry message...");
+                using var message = new Message(Encoding.UTF8.GetBytes("TestMessage"));
+                await iotClient.SendEventAsync(message);
+                await iotClient.CloseAsync();
+            }
+            finally
+            {
+                CleanupCertificates();
+
+                _logger.LogInformation("Finished.");
+            }
+        }
+
+        private ProvisioningTransportHandler GetTransportHandler()
+        {
+            return _parameters.TransportType switch
+            {
+                TransportType.Mqtt => new ProvisioningTransportHandlerMqtt(),
+                TransportType.Mqtt_Tcp_Only => new ProvisioningTransportHandlerMqtt(TransportFallbackType.TcpOnly),
+                TransportType.Mqtt_WebSocket_Only => new ProvisioningTransportHandlerMqtt(TransportFallbackType.WebSocketOnly),
+                TransportType.Amqp => new ProvisioningTransportHandlerAmqp(),
+                TransportType.Amqp_Tcp_Only => new ProvisioningTransportHandlerAmqp(TransportFallbackType.TcpOnly),
+                TransportType.Amqp_WebSocket_Only => new ProvisioningTransportHandlerAmqp(TransportFallbackType.WebSocketOnly),
+                TransportType.Http1 => new ProvisioningTransportHandlerHttp(),
+                _ => throw new NotSupportedException($"Unsupported transport type {_parameters.TransportType}"),
+            };
+        }
+
+        private string GenerateClientCertKeyPairAndCsr(string registrationId)
+        {
+            // Generate keypair
+            _logger.LogInformation($"Generating ECC P-256 {registrationId}.key file using ...");
+            string keygen = $"ecparam -genkey -name prime256v1 -out {s_dpsClientCertificateFolder}\\{registrationId}.key";
+            _logger.LogInformation($"openssl {keygen}");
```

Comment:
Perhaps indicate this is a command you are running:
```suggestion
            _logger.LogInformation($"Running: openssl {keygen}");
```